src/solidity-frontend/solidity_convert.cpp

```diff
@@ -272,8 +274,16 @@ bool solidity_convertert::get_var_decl(
     nlohmann::json init_value =
       is_state_var ? ast_node["value"] : ast_node["initialValue"];
 
+    nlohmann::json literal_type = nullptr;
+    if(
+      SolidityGrammar::get_expression_t(init_value) ==
+      SolidityGrammar::ExpressionT::Literal)
+    {
```

Comment:
There is no need for this bracket since we have just one instruction under the if-statement.